pkg/daemon/update.go

```diff
@@ -180,20 +180,30 @@ func (dn *Daemon) reconcilable(oldConfig, newConfig *mcfgv1.MachineConfig) *stri
 			msg := "Ignition Passwd Groups section contains changes"
 			return &msg
 		}
-		// check if the prior config is empty and that this is the first time running.
-		// if so, the SSHKey from the cluster config and user "core" must be added to machine config,.
 		if !reflect.DeepEqual(oldIgn.Passwd.Users, newIgn.Passwd.Users) {
+			// check if the prior config is empty and that this is the first time running.
+			// if so, the SSHKey from the cluster config and user "core" must be added to machine config.
+			glog.Infof("oldUsers: %v", oldIgn.Passwd.Users)
```

Comment:
Thank you for logging.

Reply:
retained the logging that made most sense looking at my cluster. we can add more/less in a follow-on if necessary :)